sdk/cosmosdb/cosmos/samples/v3/javascript/SasTokenAuth.js

```diff
@@ -8,7 +8,11 @@
 const path = require("path");
```

Comment:
Doesn't look like anything was changed here, just reformatted. Could you check this file out to avoid this change?